app/jobs/mail_request_job.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+class MailRequestJob < CaseflowJob
+  queue_with_priority :low_priority
+  application_attr :intake
+
+  # Purpose: performs job
+  #
+  # takes in VbmsUploadedDocument object and MailRequest object
+  #
+  # Response: n/a
+  def perform(vbms_uploaded_document, mail_request)
+    package_response = ExternalApi::PacmanService.send_communication_package_request(vbms_uploaded_document.veteran_file_number,
+                                                                                     get_package_name(vbms_uploaded_document),
+                                                                                     document_referenced(vbms_uploaded_document.id, mail_request[:copies]))
```

Comment:
```suggestion
      distribution_response = PacManService.send_distribution_request(
        package_id,
        get_recipient_hash(distribution),
        get_destinations_hash(dist_hash)
      )

```

This will allow for the job to be executed locally.